### DISCUSSION TITLE
Turbopack: Ability to Disable Certain Warnings

### DISCUSSION THREAD

**Discussion by ivanskodje** [vercel/next.js#71983] [Ideas]
### Goals

We would like the ability to disable nagging warning messages caused by Turbopack

### Background

Currently whenever I run my app locally for local development, it is logging a lot of warnings on every single page I visit, since I use AVIF logos:

```
 POST / 200 in 102ms
 ⚠ ./public/res/logo-large.avif
AVIF image not supported
This version of Turbopack does not support AVIF images, will emit without optimization or encoding


 ⚠ ./public/res/logo.avif
AVIF image not supported
This version of Turbopack does not support AVIF images, will emit without optimization or encoding
``` 

I am sorry, but Turbopack's inability to handle AVIF images is **not** my problem, and I do not want to get nagged about it. 

We have not found any documentation or information about how to disable this. This is in fact so troublesome that we have removed turbopack from the projects, because the nag is so bothersome and makes it harder to keep track of logging. 

Essentially, this prevents us from working properly.

### Proposal

Give us an option to disable the warning permanently as the warning

**Comment by mingster** [comment]
+1

**Comment by SUMIT-WEBDEV** [comment]
+1

**Comment by kyoto21** [comment]
+1

**Comment by styxlab** [comment]
I am even getting this warning message on WEBP images, i.e.
```
 ⚠ ./app/images/articles/test/test.webp
AVIF image not supported
This version of Turbopack does not support AVIF images, will emit without optimization or encoding
```
Warning message is flooding the console, really annoying.

**Comment by styxlab** [comment]
Does this request need to be transferred to an issue in order to be considered?

**Comment by YungBricoCoop** [comment]
+1

**Comment by jens-duttke** [comment]
I use this workaround for these annoying warnings:

Just add the following code to the beginning of your `next.config.js` or `next.config.ts`:
```js
const originalStderrWrite = process.stderr.write.bind(process.stderr);

process.stderr.write = ((chunk, ...args) => {
	if (chunk.toString().includes('AVIF image not supported')) {
		return true;
	}

	return originalStderrWrite(chunk, ...args);
});
```

Tested with Next.js v16.0.1
